pkg/blockstorage/azure/client.go

```diff
@@ -98,9 +93,24 @@ func NewClient(ctx context.Context, config map[string]string) (*Client, error) {
 	}, nil
 }
 
+func GetEnvironment(config map[string]string) (*azure.Environment, string, error) {
```

Comment:
It doesn't look like the 2nd returned string is being used.

Reply:
hi @ihcsim , it was going to be used in a downstream caller. However, after some more thought into the design, this approach is more complicated than necessary. I am switching to https://github.com/kanisterio/kanister/pull/1466 instead